src/libs/NetworkResponseManager.js

```diff
@@ -0,0 +1,71 @@
+import _ from 'underscore';
+
+function handleDefaultError(jsonCode, response) {
+    // @TODO do some default handling
+}
+
+const NetworkResponseManager = {
+    commandHandlers: {},
+    subscribe(commandName) {
+        const connection = {
+            onSuccess: null,
+            onHandle: null,
+        };
+
+        this.commandHandlers[commandName] = this.commandHandlers[commandName] || [];
+        this.commandHandlers[commandName].push(connection);
+
+        connection.done = (successCallback) => {
+            connection.onSuccess = successCallback;
+            return connection;
+        };
+
+        connection.handle = (jsonCodes, callback) => {
+            connection.onHandle = [jsonCodes, callback];
+            return connection;
+        };
+
+        connection.cancel = () => {
```

Comment:
What would this be used for?

Reply:
Might not be necessary at all - I just jacked this code from a very minimal PubSub implementation to make this "complete". Probably these subscriptions would be like our many `Onyx.connect()` calls that do not need to be explicitly "unsubscribed" or "disconnected".